backend/routes_kit_transfers.py

```
@@ -65,22 +66,90 @@ def create_transfer():
+                    part_number=source_item.part_number if source_item else None
+                ).first()
+
+                if dest_item:
+                    # Add to existing expendable
+                    dest_item.quantity += quantity
+                else:
+                    # Create new expendable in destination kit
+                    # Get first available box or create in loose
+                    dest_box = dest_kit.boxes[0] if dest_kit.boxes else None
+                    if not dest_box:
+                        raise ValidationError('Destination kit has no boxes')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)